packages/paypal-fastlane-integration/src/is-fastlane-window.ts

```diff
@@ -0,0 +1,25 @@
+export interface FastlaneWatermarkComponent {
+    FastlaneWatermarkComponent: (
+        options?: FastlaneWatermarkOptions,
+    ) => Promise<FastlanePrivacySettings>;
+}
+
+interface FastlaneWatermarkOptions {
+    includeAdditionalInfo: boolean;
+}
+
+export interface FastlanePrivacySettings {
+    render(container: string): void;
+}
+
+declare global {
```

Comment:
I don't really sure that we need global declaration

Reply:
removed